style/javascript.md

```diff
@@ -552,3 +553,52 @@ Yes:
 ```js
 $(".some-class span").hide();
 ```
+
+#### Don't use Underscore
+
+We use ES6/7 which includes many of the features of Underscore.js! Using Underscore should be avoided in favor of these native language features.
```

Comment:
Lodash has over 200 methods and Underscore has over 115 or so, far more than the handful of ES6/ES7 methods. The line above suggests that ES6/7 can replace most which is not the case, ES6/7 can replace a handful of narrow use cases.